src/Microsoft.AspNet.Mvc.Core/ConsumesAttribute.cs

```diff
@@ -30,6 +32,15 @@ public ConsumesAttribute(string contentType, params string[] otherContentTypes)
                 throw new ArgumentNullException(nameof(contentType));
             }
 
+            // We want to ensure that the given provided content types are valid values, so
+            // we validate them using the semantics of MediaTypeHeaderValue.
+            MediaTypeHeaderValue.Parse(contentType);
+
+            for (var i = 0; i < otherContentTypes.Length; i++)
+            {
+                MediaTypeHeaderValue.Parse(otherContentTypes[i]);
+            }
```

Comment:
Ah, sorry. I missed the `Parse()` that used to be on [line 175](https://github.com/aspnet/Mvc/pull/3862/files#diff-1b5675f1b083c83b1f98dd3e87cf08f6L175).